include/ck/utility/flush_icache.hpp

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2024, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include <hip/hip_runtime.h>
+
+namespace ck {
+static __global__ void flush_icache()
+{
+#pragma unroll
+    for(int i = 0; i < 100; i++)
```

Comment:
Can we remove this unroll? it will generate ~2000line of code in asm file. actually #pragma unroll + dynamic for loop is very similar with static for loop

Reply:
done